domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityACLValidatorImplTest.java

```diff
@@ -57,7 +57,7 @@ public void shouldHaveValidationErrorWhenUserNotFound() {
         assertThat(result.getValidationErrors().size(), is(1));
         assertThat(result.getValidationErrors().get(0), instanceOf(CaseFieldEntityInvalidUserRoleValidationError.class));
         assertThat(result.getValidationErrors().get(0).getDefaultMessage(), is(
-            "Invalid UserRole for case type 'case_type', case field 'case_field'"));
+            "Invalid UserRole null for case type 'case_type', case field 'case_field'"));
```

Comment:
just out of curiosity will we ever get `Invalid UserRole null` in reality? I don't think am I correct?

Reply:
If the user role that we have passed doesn't exists in the system then we will get null value here

Reply:
👍